assets/src/scripts/components/hydrograph/method-picker.js

```diff
@@ -33,6 +33,11 @@ export const drawMethodPicker = function(elem) {
                     .text(method.methodDescription ? `${method.methodDescription}` : 'None')
                     .node().value = method.methodID;
             });
+            if (methods.length > 1) {
```

Comment:
Consider reducing this if clause to a single statement: ```pickerContainer.property('hidden', methods.length <= 1);```

Reply:
Oh nice, done!